src/sentry/static/sentry/app/plugins/components/settings.jsx

```diff
@@ -109,6 +109,10 @@ class PluginSettings extends PluginComponentBase {
     let hasChanges = !underscore.isEqual(this.state.initialData, this.state.formData);
 
     let data = this.state.rawData;
+
+    if (!this.props.hasConfigOptions) {
```

Comment:
is there a reason doing something like `if (!(this.state.fieldList || []).length) {` won't work here? i think that should only have actual form fields that need to be configured